src/box/box.cc

```diff
@@ -2429,10 +2429,8 @@ box_make_bootstrap_leader_nongraceful(void)
 int
 box_make_bootstrap_leader(bool graceful)
 {
-	/* Bootstrap strategy is read by the time instance uuid is known. */
```

Comment:
Please leave a comment about `bootstrap_strategy_invalid` meaning. That it means we haven't read the config yet.